### ISSUE TITLE
Shipwright explodes if Docker build outputs unicode characters.

### ISSUE THREAD

**Issue by spudfkc** [6si/shipwright#21]
When trying to run shipwright I am getting the following error:
```bash
Traceback (most recent call last):
  File "/home/********/envs/slave/bin/shipwright", line 9, in <module>
    load_entry_point('shipwright==0.2.3', 'console_scripts', 'shipwright')()
  File "/home/********/envs/slave/local/lib/python2.7/site-packages/shipwright/cli.py", line 195, in main
    show_fn(writer(event))
  File "/home/********/envs/slave/local/lib/python2.7/site-packages/shipwright/cli.py", line 228, in highlight_
    print(color_fn(name) + " | " + msg)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2018' in position 39: ordinal not in range(128)
```

This specific line that was causing it was a `ln -sv` command which outputs unicode characters (not quite sure why `ln` needs to output unicode, but that is besides the point).

I worked around this by just stripping out non-ascii characters, but I'm not sure if that's the best solution for this problem.
this is `cli.py` line 228 on the latest release of shipwright - 0.2.3
```python
  def highlight_(msg):
    msg = ''.join([i if ord(i) < 128 else ' ' for i in msg])
    print(color_fn(name) + " | " + msg)
```

**Comment by graingert** [comment]
@spudfkc I think this is fixed in Python 3.5 ;)

Can you right a failing test please?

**Comment by graingert** [comment]
@spudfkc I'm closing this as I think it's fixed, if not can you open a PR with a failing test/your fix.
